pyproject.toml

```diff
@@ -48,6 +48,7 @@ ignore = [
     "dev/**",
     "docker/**",
     "openshift/**",
+    "galaxy_ng/app/static/**",
```

Comment:
@rochacbruno I mistakenly said you didn't need this, but I was wrong, sorry about that!
checkmanifest compares the package with the VCS, and the gitignore makes them diverge, so indeed you need to ignore it here